config/src/main/java/com/quorum/tessera/config/ServerConfig.java

```diff
@@ -53,7 +46,10 @@
     @XmlElement(name = "cors")
     private CrossDomainConfig crossDomainConfig;
 
-    public ServerConfig(final AppType app,
+    @XmlElement private Integer syncInterval;
```

Comment:
If you change this to be `@XmlElement private int syncInterval = 5000;` and add the @Min(value = 1L) validation annotation, then it should be set up to avoid nulls or negative values (and reduce checks elsewhere).

What do you think?

Reply:
don't use primitive for config values please. But sync interval doesn't belong on a server config. lets not waste time commenting here and take conversions offline or go in and make changes directly.

Reply:
Sure, can avoid using a primitive. We can still set the default value and add the validation rule for the same effect (wherever it may end up).